### PR TITLE
[Logging] add client key and certificate for logging fluentd

### PR DESCRIPTION
**Pull request by aiwantaozi**
Problem:
advance mode have support client key and certificate for fluentd logging
target but normal mode not yet

Solution:
add client key and certificate for fluentd

types:
https://github.com/rancher/types/pull/739

Issue:
https://github.com/rancher/rancher/issues/18438

Integration test pr:
https://github.com/rancher/validation/pull/161

### CODE REVIEW

**Comment by orangedeng** [comment]
LGTM, should merge this PR after types PR is merged.

**Comment by aiwantaozi** [comment]
rebase to resolve conflict

**Comment by orangedeng** [comment]
need rebase again

File: tests/validation/tests/v3_api/test_logging.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+username = "user1"
+password = "my_password"
+shared_key = "my_shared_key"
+weight = 100
+
+
+def test_fluentd_target(request):
+    cluster = namespace["cluster"]
+    cluster_logging = create_cluster_logging(fluentd_target_without_ssl())
+    request.addfinalizer(lambda: delete_cluster_logging(cluster_logging))
```
Comment (mrajashree) [inline tests/validation/tests/v3_api/test_logging.py]: We can use remove_resource from conftest.py

Comment (aiwantaozi) [reply tests/validation/tests/v3_api/test_logging.py]: Hi @mrajashree remove_resource is under integration test, these test cases are under validation test, is this function available in validation test?

**Comment by aiwantaozi** [comment]
rebase to resolve conflict

**Comment by cjellick** [comment]
types pr merged

**Comment by aiwantaozi** [comment]
rebased, thanks

**Comment by alena1108** [comment]
lgtm

**Pull request merged**